test/rules-engine/sssi.test.js

```diff
@@ -0,0 +1,60 @@
+describe('Rules engine rule test', () => {
+  let rulesEngine
+  const parcelsTestData = require('./test-data/parcels-sssi.json')
+  const testRules = require('./test-data/rules-sssi.json')
+  const successEvent = jest.fn()
+  const eventType = 'sssi'
+
+  beforeAll(async () => {
+    rulesEngine = require('../../server/rules-engine')
```

Comment:
Similar here - you could just move the require to the top of the block and make rulesEngine a const. Also, no need for the const.

Reply:
Agree. Changed.